.gitignore

```diff
@@ -3,3 +3,4 @@
 npm-debug.log
 /package-lock.json
 tmp/
+.DS_Store
```

Comment:
This should rather be configured in user global ignore (this file is intentionally pretty simple, and contains only ignores that are directly related to project.

See: https://eshlox.net/2017/08/04/git-global-gitignore-file